source/userguide/kb.rst

```diff
@@ -701,3 +701,88 @@ In the example below, we search for restaurants whose names best match ``firetra
 .. note::
 
    We can set the ``size`` parameter of the :meth:`execute()` method to specify the maximum number of records.
+
+
+Dealing with unstructured text
+------------------------------
+The knowledge bases described in the previous sections are comprised of clearly defined data types whose pattern makes them easily searchable. We call them structured data.
+
+In this section we describe how to use the Question Answerer on knowledge bases with long, free-form text or unstructured data. For example, a company's Human Resource (HR) team could have documents about general policies of the company. We can use this data to develop a HR assistant that automatically answers any policy related questions.
+
+To use the question answerer on unstructured text, we would follow the same steps mentioned before to prepare, index and load the knowledge base.
+Here is an example of what the knowledge base can look like for the HR assistant usecase. It consists of frequently asked question and answer pairs.
+
+.. code:: python
+
+	from mindmeld.components import QuestionAnswerer
+	qa = QuestionAnswerer(app_path='hr_assistant')
+	qa.load_kb(app_namespace='hr_assistant', index_name='faq_data', data_file='hr_assistant/data/faq_data.json')
+  qa.get(index='faq_data')
```

Comment:
```suggestion
 qa.get(index='faq_data')
```

Reply:
I think it should instead have an additional space to align with the previous lines.